internal/proxy/task.go

```diff
@@ -482,7 +488,7 @@ func (dct *describeCollectionTask) Execute(ctx context.Context) error {
 		dct.result.Aliases = result.Aliases
 		dct.result.Properties = result.Properties
 		for _, field := range result.Schema.Fields {
-			if field.FieldID >= common.StartOfUserFieldID {
+			if field.FieldID >= common.StartOfUserFieldID || field.IsDynamic {
```

Comment:
Since the fieldID of the dynamic field is greater than common.StartOfUserFieldID, is the second criterion unnecessary?